javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,143 @@
+//Flight booking fullname function
+function getFullname(firstname, surname, gender, useFormalName) {
+	if (gender === "male" && useFormalName === true) {
```

Comment:
instead can use 'switch case'. try otherwise this is also fine.

Reply:
ok. i will try with switch case. thank you